app/models/owner.rb

```diff
@@ -18,7 +18,7 @@ class Owner < ActiveRecord::Base
   serialize :feature_switches
 
   # Supporters that are on a particular plan
-  scope :supporters, ->(plan) { where(stripe_plan_id: plan.stripe_plan_id) }
+  scope :supporters, ->(plan) { where(stripe_plan_id: plan.stripe_plan_id).order("RAND()") }
```

Comment:
>Note, this change uses an MySQL only option, but as Morph.io seems pretty closely bound with MySQL, that seemed fine.

If you do decide to make the change I've suggested above, I'd also recommend doing it in Ruby since it will be easier to read. Performance won't be an issue so clarity should trump that in this case :sparkles: